assignments/arrays.js

```diff
@@ -63,32 +65,81 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(inventory[32])
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+
+for(let i = 0; i < inventory.length; i++){
+  if(inventory[i].id === 33) {
+    //string.length = 50
+    //Foreach iteration: i = ?      i < inventory.length   i++   if(inventory[i].id === 33)           
+    //1st                   0                         0 < 50 true           1      1 === 33 false                        
+    //2nd                 1                          1 < 50 true           2      2 === 33 false                        
+    //3rd                  2                          2 < 50 true           3      3 === 33 false 
+    //33rd                32                        33< 50 true          34  33 === 33 true              == > console.log                          
+    //     }
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
+  }
+}
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// let lastCar = 0;
+// console.log();
+
+// let lastCar = 0;
+for (let i = 0; i < inventory.length; i++){
+  if(inventory[i] === inventory[inventory.length -1]) {
+    // 49 ===  49  true
+    console.log(`${inventory[i].car_make} ${inventory[i].car_model}`)
+  }
+}
+let lastCar = inventory.length -1;
+console.log(`${inventory[lastCar].car_make} ${inventory[lastCar].car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for(let i = 0; i < inventory.length; i++){
+  carModels.push(inventory[i].car_model)
+  // car model list is added(push) to the empty array carModels
+}
+carModels.sort();
+// car model list is sort alphabetical order with sort() method
+console.log(carModels);
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for(let i = 0; i < inventory.length; i++){
+  carYears.push(inventory[i].car_year);
+  // car year list is added(push) to the empty array carYears 
+}
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars = [];
-console.log(); 
+for(let i = 0; i < inventory.length; i++){
+  if(carYears[i] < 2000){
```

Comment:
Awesome that you used the existing `carYears` array you created in Challenge 4! 🔥 🔥